sdk/servicebus/azure-servicebus/azure/servicebus/_common/message.py

```diff
@@ -967,6 +969,27 @@ def dead_letter_source(self):
                 pass
         return None
 
+    @property
+    def message_state(self):
```

Comment:
I was wondering should `ServiceBusMessage` also support this property with value always being `ACTIVE`?

could you check whether .NET just has the property for received message?

Reply:
.net has it only in the received message  - also , we can add it later if necessary (but not remove it)

Reply:
sounds good!